rootcmd/root.go

```diff
@@ -96,6 +96,12 @@ func assignPersistentFlags(cmd *cobra.Command, flags *config.GlobalFlagValues) {
 	cmd.PersistentFlags().BoolVar(&flags.Verbose, "verbose", false, "verbose output")
 	cmd.PersistentFlags().BoolVar(&flags.NoColor, "no-color", false, "disable colors")
 	cmd.PersistentFlags().BoolVar(&flags.NoAutocompletion, "no-completion", false, "disable completion [interactive mode]")
+	cmd.PersistentFlags().StringVar(&flags.LogFile, "log-file", "", "direct logs to specified file")
+	cmd.PersistentFlags().StringVar(&flags.LogLevel, "log-level", "", "(default 'error') write logs with specified or higher level of importance")
+	cmd.RegisterFlagCompletionFunc("log-level", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return config.ValidLogLevels, cobra.ShellCompDirectiveDefault
+	})
```

Comment:
This suggests possible values in _non-interactive_ mode when you press tab (assuming you have typed `hzc --log-level`).

In this case, it does not affect the interactive mode, since we remove all the top level flags from suggestions and we ignore them if they are provided. So user can not change the log level dynamically.